mvc/src/main/java/nextstep/mvc/view/JsonView.java

```diff
@@ -1,13 +1,46 @@
 package nextstep.mvc.view;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-
+import java.io.BufferedWriter;
+import java.io.OutputStreamWriter;
 import java.util.Map;
+import java.util.Map.Entry;
+import java.util.stream.Collectors;
+import nextstep.web.support.MediaType;
 
 public class JsonView implements View {
 
+    private final ObjectMapper objectMapper;
+
+    public JsonView() {
+        this.objectMapper = new ObjectMapper();
+    }
+
     @Override
-    public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response) throws Exception {
+    public void render(final Map<String, ?> model, final HttpServletRequest request, HttpServletResponse response)
+            throws Exception {
+        final var modelToWrite = getModelToWrite(model);
+        final var responseBody = objectMapper.writeValueAsString(modelToWrite);
+        response.setContentType(MediaType.APPLICATION_JSON_UTF8_VALUE);
+        response.setContentLength(responseBody.getBytes().length);
+
+        try (final var bufferedWriter = new BufferedWriter(new OutputStreamWriter(response.getOutputStream()))) {
+            bufferedWriter.write(responseBody);
+            bufferedWriter.flush();
+        }
+    }
+
+    private Object getModelToWrite(final Map<String, ?> model) {
+        if (model.size() == 1) {
+            return model.entrySet()
+                    .stream()
+                    .map(Entry::getValue)
+                    .collect(Collectors.toList())
+                    .get(0);
```

Comment:
```suggestion
       if (model.size() == 1) {
            return new ArrayList<>(model.values());
        }
```
현재 코드를 봤을 때  Entry에서 value만 가지고와 List로 만들어주는 형태인 것 같습니다. 그래서 위와 같이 만들어줄 수도  있을 것 같아요 ㅎㅎ